components/mediation-admin/org.wso2.carbon.localentry/src/main/java/org/wso2/carbon/localentry/service/LocalEntryAdmin.java

```diff
@@ -238,6 +239,26 @@ public boolean addEntry(String ele) throws LocalEntryAdminException {
         return false;
     }
 
+    /**
+     * Add a entry into the synapseConfiguration
+     *
+     * @param ele          - Entry object to be added as an OMElement
+     * @param tenantDomain - Tenant domain
+     * @return whether the operation is successfull or not
+     * @throws LocalEntryAdminException if a Entry exists with the same name or if the
+     *                                  element provided is not a Entry element
+     */
+    public boolean addEntryForTenant(String ele, String tenantDomain) throws LocalEntryAdminException {
```

Comment:
"ele" is not a readable variable name. Shall we have a proper name?

Reply:
Updated